src/operator/spatial_transformer.cc

```diff
@@ -27,8 +27,8 @@
 
 namespace mshadow {
 template<typename DType>
-bool between(DType value, int lowerBound, int upperBound) {
-  return (value >= lowerBound && value <= upperBound);
+static MSHADOW_CINLINE bool between(const DType value, const DType lowerBound, const DType upperBound) {
```

Comment:
why static?

Reply:
because this function is used only in this file. static avoids link collisions with other object files which may have the same function defined.  Since its prototype is not declared outside this file, then by convention, it should be declared as static.

Reply:
note that the writer put a "copy" of this function in several files

Reply:
https://github.com/apache/incubator-mxnet/blob/master/src/operator/bilinear_sampler.cc#L30

Just to add context, similar function is defined again in the above file. There is no difference in the functions but is defined again. Should we change them to use the same function?

Reply:
Probably, but I think that's out of the scope of this change.